src/act.other.cpp

```diff
@@ -4637,22 +4637,22 @@ ACMD(do_syspoints) {
     }
 
     // Mail the victim.
-    snprintf(buf, sizeof(buf), "You have been %s %d system points for %s.\r\n",
+    snprintf(buf, sizeof(buf), "You have been %s %d system points for%s\r\n",
```

Comment:
Why was the space removed? Is it outputting with two spaces currently?

Reply:
Nope, that was a mistake on my part. Corrected.